exasol_sphinx_github_pages_generator/deploy_github_pages.py

```diff
@@ -49,5 +49,9 @@ def deploy_github_pages(argv=sys.argv[1:]):
             deployer.clean_worktree(original_workdir)
 
 
-if __name__ == "__main__":
+def main():
     deploy_github_pages(sys.argv[1:])
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
Make sure you get a clean exit and report the exit code, this will help other to integrate into CI & Co.

```suggestion
    sys.exit(main())
```

Reply:
done